packages/portal-proto/src/components/Modals/SetModals/UpdateFiltersButton.tsx

```diff
@@ -69,7 +72,7 @@ const UpdateCohortButton: React.FC<UpdateCohortButtonProps> = ({
       disabled={disabled}
       onClick={() => {
         if (ids.length > 1) {
-          createSet({ values: ids });
+          createSet({ values: ids, intent: "user", set_type: "frozen" });
         } else {
```

Comment:
<img width="842" alt="Screenshot 2024-04-05 at 2 58 01 PM" src="https://github.com/NCI-GDC/gdc-frontend-framework/assets/4624053/94322a6d-45d7-45d5-9e0a-ea0311dfd313">

Reply:
ephemeral - the user may not end up saving that cohort.

torn on 'user' or 'portal'.

Reply:
I feel it's portal

Reply:
I say **user**: The action is Save Set, something the user is asking for

Reply:
@craigrbarnes the submit just filters the current cohort and doesn't create a set they can use later (it doesn't show up in Manage Sets). the Save Set action will create a set they can use later.